packages/@angular/cli/blueprints/ng2/files/__path__/tsconfig.app.json

```diff
@@ -0,0 +1,28 @@
+{<% if (ng4) { %>
+  "extends": "<%= relativeRootPath %>/tsconfig.json",
+  "compilerOptions": {
+  "lib": [
+    "es2016",
+    "dom"
+  ],<% } else { %>
+  "compilerOptions": {
+    "sourceMap": true,
+    "declaration": false,
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "lib": [
+      "es2016",
+      "dom"
+    ],<% } %>
+    "outDir": "<%= relativeRootPath %>/out-tsc/app",
+    "target": "es5",
+    "module": "es2015",
+    "baseUrl": "",
+    "types": []
```

Comment:
@filipesilva A bit late question but why declaring `types` manually is better than having one global `typeRoots` (as there was before)?
```json
"typeRoots": [
    "../node_modules/@types"
]
```

Reply:
This was hard to dig out, but thanks for clarifying. 👍